css/index.css

```diff
@@ -1,3 +1,399 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  margin: 0;
+  padding: 0;
+  vertical-align: baseline;
+}
 
+/* HTML5 display-role reset for older browsers */
+
+article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+}
+
+ol, ul {
+  list-style: none;
+}
+
+blockquote, q {
+  quotes: none;
+}
+
+blockquote:before, blockquote:after, q:before, q:after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+
+* {
+  box-sizing: border-box;
+}
+
+html, body {
+  --gray: rgba(0, 0, 0, 0.5);
+  --lightgray: rgba(127, 127, 127, 0.05);
+  --main-color: black;
+  --main-font: 'Titillium Web', sans-serif;
+  --main-width: 876px;
+  --top-nav-height: 5rem;
+  /* Media queries */
+  color: var(--main-color);
+  font-family: var(--main-font);
+  font-size: 16px;
+  height: 100%;
+}
+
+@media(max-width: 800px) {
+  html, body {
+    --main-width: 95vw;
+    --top-nav-height: 12rem;
+  }
+}
+
+@media(max-width: 500px) {
+  html, body {
+    --main-width: 100vw;
+  }
+}
+
+h1, h2, h3, h4, h5 {
+  font-family: 'Bangers', cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+/*******/
+
+/* NAV */
+
+/*******/
+
+img {
+  max-width: 100%;
+}
+
+.container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.top-nav-container {
+  align-items: center;
+  background: #fffffffa;
+  display: flex;
+  height: var(--top-nav-height);
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+@media (max-width: 500px) {
+  .top-nav-container {
+    position: relative;
+    height: auto;
+  }
+}
+
+.top-nav {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  width: var(--main-width);
+}
+
+@media (max-width: 800px) {
```

Comment:
Nice job putting these media queries just below their desktop styling. You can also put the media queries at the very bottom of your CSS, but doing it the way you have here is generally easier to read.